transports/azure-service-bus/legacy/configuration/full_migration_asb_[9.1,).partial.md

```diff
@@ -0,0 +1,5 @@
+#### Migration options
+
+Migration API to turn enable [migration](/transports/azure-service-bus/legacy/migration.md) from Endpoint-oriented topology to Forwarding topology, migration feature
```

Comment:
It will appear as a sub-section of https://docs.particular.net/transports/azure-service-bus/legacy/configuration/full#configuring-the-topology-endpoint-oriented-topology